Formula/kubebuilder.rb

```diff
@@ -0,0 +1,40 @@
+class Kubebuilder < Formula
+  desc "SDK for building Kubernetes APIs using CRDs"
+  homepage "https://github.com/kubernetes-sigs/kubebuilder"
+  url "https://github.com/kubernetes-sigs/kubebuilder.git",
+      :tag      => "v2.0.0-alpha.1",
```

Comment:
This doesn't seem to match: https://docs.brew.sh/Acceptable-Formulae#stable-versions

Reply:
The latest stable release uses a different package manager as well as a different install mechanism. Is there a tag that can be added to indicate its a alpha release?

Reply:
No. My vote is we wait for upstream to mint a proper release. The other option is keep this in your back pocket for when they do, and submit the older release, using the older installation, although I'd rather not go that route.

You can also start your own tap for now, then re-submit to core once upstream has a stable release.

Reply:
We'll be releasing a stable release soon (the plan was to let it soak for a week or two after KubeCon), so you shouldn't have to wait long :-)